### PR TITLE
Fix/tab bar resize callback is not fired when a new tab added

### PR DESCRIPTION
**Pull request by Sarin-Udompanish**
## Description
Resize observer always observed on the root element but the contents of tab-bar are inside the content part so `resizeCallback` will not fired when new contents are modified or updated.

Fixes # (issue)
Resize callback is not fired when a new tab added

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: packages/elements/src/time-picker/__snapshots__/TimePicker.md [hunk 1]
```diff
@@ -205,7 +205,6 @@
 
 ```html
 <ef-number-field
-  aria-label="1 hour"
```
Comment (wsuwt) [inline packages/elements/src/time-picker/__snapshots__/TimePicker.md:208]: Timepicker ?

**Comment by wsuwt** [comment]
LGTM. Did you test resizing on IE11 ?

**Comment by Sarin-Udompanish** [comment]
> 

Yes i did. It work fine.

**Comment by Sarin-Udompanish** [comment]
> LGTM. Did you test resizing on IE11 ?

Yes i did. It work fine.

**Review by wsuwt — DISMISSED** [review]

**Review by wsuwt — APPROVED** [review]

**Pull request merged**